pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -87,7 +91,7 @@ func NewEtcdCertSignerController(
 	eventRecorder = eventRecorder.WithComponentSuffix("etcd-cert-signer-controller")
 	cmInformer := kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().ConfigMaps()
 	cmLister := cmInformer.Lister()
-	cmGetter := v1helpers.CachedConfigMapGetter(kubeClient.CoreV1(), kubeInformers)
+	cmGetter := kubeClient.CoreV1()
```

Comment:
btw, I removed the caching here because it was causing flaky tests - that should also help with the recert command...

Reply:
actually, funny enough: because the cache is synced on the same informer the results are always consistent :rofl: 

Otherwise the informer might be lagging behind what the client getter would return.